.travis/apisix_cli_test/test_main.sh

```diff
@@ -705,6 +707,12 @@ if [ ! $? -eq 0 ]; then
     exit 1
 fi
 
+grep 'server_names_hash_bucket_size 128;' -A 2 conf/nginx.conf | grep "configuration snippet ends" > /dev/null
```

Comment:
Would be better if we also check there is a '}' in the output, like:
grep 'server_names_hash_bucket_size 128;' -A 3 conf/nginx.conf | grep "}" > /dev/null